shared/params/config.go

```diff
@@ -9,81 +9,112 @@ import (
-	DomainRandao      uint64 // DomainRandao defines the BLS signature domain for randao verification.
-	DomainTransfer    uint64 // DomainTransfer defines the BLS signature domain for transfer verification.
+	ShardCount               uint64 // ShardCount is the number of shard chains in Ethereum 2.0.
+	TargetCommitteeSize      uint64 // TargetCommitteeSize is the number of validators in a committee when the chain is healthy.
+	MaxIndicesPerAttestation uint64 // MaxIndicesPerAttestation is used to determine how many validators participate in an attestation.
+	MinPerEpochChurnLimit    uint64 // MinPerEpochChurnLimit is the minimum amount of churn allotted for validator rotations.
+	ChurnLimitQuotient       uint64 // ChurnLimitQuotient is used to determine the limit of how many validators can rotate per epoch.
+	ShuffleRoundCount        uint64 // ShuffleRoundCount is used for retrieving the permuted index.
+	// TODO(2307): Remove deprecated fields
+	MaxBalanceChurnQuotient uint64 // MaxBalanceChurnQuotient is used to determine how many validators can rotate per epoch.
+	// Deprecated: No longer in use.
+	BeaconChainShardNumber uint64 // BeaconChainShardNumber is the shard number of the beacon chain.
+	// Deprecated: Do not use.
+	MaxIndicesPerSlashableVote uint64 // MaxIndicesPerSlashableVote is used to determine how many validators can be slashed per vote.
+	// Deprecated: Do not use.
+	MaxExitDequeuesPerEpoch uint64 // MaxWithdrawalsPerEpoch is the max withdrawals can happen for a single epoch.
+	// Deprecated: Do not use.
 
 	// Deposit contract constants.
 	DepositContractAddress   []byte // DepositContractAddress is the address of the deposit contract in PoW chain.
 	DepositContractTreeDepth uint64 // Depth of the Merkle trie of deposits in the validator deposit contract on the PoW chain.
 
-	// Gwei Values
-	MinDepositAmount           uint64 // MinDepositAmount is the maximal amount of Gwei a validator can send to the deposit contract at once.
-	MaxDepositAmount           uint64 // MaxDepositAmount is the maximal amount of Gwei a validator can send to the deposit contract at once.
-	EjectionBalance            uint64 // EjectionBalance is the minimal GWei a validator needs to have before ejected.
+	// Gwei value constants.
+	MinDepositAmount     uint64 // MinDepositAmount is the maximal amount of Gwei a validator can send to the deposit contract at once.
+	MaxDepositAmount     uint64 // MaxDepositAmount is the maximal amount of Gwei a validator can send to the deposit contract at once.
+	EjectionBalance      uint64 // EjectionBalance is the minimal GWei a validator needs to have before ejected.
+	HighBalanceIncrement uint64 // HighBalanceIncrement is used for converting the high balance into the low balance for validators.
+	// TODO(2307): Remove deprecated fields
 	ForkChoiceBalanceIncrement uint64 // ForkChoiceBalanceIncrement is used to track block score based on balances for fork choice.
+	//Deprecated: Do not use.
```

Comment:
What does this go to?

Reply:
This was changed in 
https://github.com/ethereum/eth2.0-specs/commit/0759e170a7faafca0cee7b9b8929b0c6e15bcf77

Reply:
Then lets remove it? it is a new line of code that does to nothing